terraform/environments/nomis/ansible/group_vars/server_type_base.yml

```diff
@@ -0,0 +1,4 @@
+---
+ansible_python_interpreter: /usr/local/bin/python3.9
```

Comment:
might run into a problem here on the rhel 6.10 ami's as they're using python 3.6 only?

Reply:
Probably it should be baseimage_7_9 or some such, but basically all EC2 instances with the same server-type tag will have the same configuration, i.e.the same AMI image.  So say a "nomis-weblogic" server-type would have 3.6 set

Reply:
Ah, great 👍